### DISCUSSION TITLE
Error while pushing

### DISCUSSION THREAD

**Discussion by OverLimitsGames** [community/community#51680] [Pull Requests]
### Select Topic Area

Question

### Body

Hello everyone, 
I'm trying to push to the origin a commit I made earlier (its size is 1.3GB) but I get the following error all the time:
"Error - The remote disconnected. Check your internet connection and try again."

I've already tried increasing the LFS Activity timeout and the global HTTP low-Speed Limit and Time, as suggested in a previous discussion, but no benefit.

**Comment by airtower-luna** [comment]
That sounds like your commit is *way* too large, GitHub has limits on file sizes. See: https://docs.github.com/en/repositories/working-with-files/managing-large-files/about-large-files-on-github
